### PR TITLE
Added support for general.timestampPath

### PR DESCRIPTION
**Pull request by jeremyharris**
Depends on git@github.com:jeremyharris/asset_compress.git

Well, sort of. It'll work just fine but won't parse constants until the above is released. This can be on hold until I make any required composer updates.

### CODE REVIEW

File: src/Factory.php [hunk 1]
```diff
@@ -32,7 +32,7 @@ class Factory extends BaseFactory
      */
     public function writer($path = TMP)
     {
-        return parent::writer($path);
+        return parent::writer($this->config->get('general.timestampPath') ?: $path);
```
Comment (markstory) [inline src/Factory.php]: One thing I realized after merging the mini-asset change, is that this path also moves the temporary files created in development mode.

Comment (jeremyharris) [inline src/Factory.php]: What temporary files? I'm not seeing any in my setup, but perhaps I'm missing a configuration setting somewhere. I'll run some tests.

Comment (markstory) [inline src/Factory.php]: Nevermind, I thought the AssetWriter was still used for development build files but it isn't.

**Comment by jeremyharris** [comment]
Bumped dependency and fixed some unrelated CS errors that were causing the build to fail because CS.

**Comment by jeremyharris** [comment]
Thanks a ton 😄

**Pull request merged**